central/networkgraph/service/env.go

```diff
@@ -0,0 +1,12 @@
+package service
+
+import (
+	"strconv"
+
+	"github.com/stackrox/rox/pkg/env"
+)
+
+var (
+	defaultMaxNumberOfDeploymentsInGraph = 2000
+	maxNumberOfDeploymentsInGraphEnv     = env.RegisterSetting("ROX_MAX_DEPLOYMENTS_NETWORK_GRAPH", env.WithDefault(strconv.Itoa(defaultMaxNumberOfDeploymentsInGraph)))
```

Comment:
You can use `env.RegisterIntegerSetting`, that will save you some code in the place where you use this as well 🙂

Reply:
I didn't know about this feature. Thanks for pointing it out!